spec/models/manageiq/providers/google/cloud_manager/refresher_spec.rb

```diff
@@ -183,6 +192,32 @@ def assert_specific_floating_ips
     expect(unassigned_floating_ip.network_port).to eql(nil)
   end
 
+  def assert_specific_load_balancer
+    lb = LoadBalancer.where(:name => "foo-lb-forwarding-rule").first
+
+    expect(lb).to have_attributes(
+      :name    => "foo-lb-forwarding-rule",
+      :ems_ref => "1778652908557222005",
+      :type    => "ManageIQ::Providers::Google::NetworkManager::LoadBalancer"
+    )
+    expect(lb.load_balancer_listeners.first).to have_attributes(
+      :name                     => "foo-lb-forwarding-rule",
+      :ems_ref                  => "1778652908557222005",
+      :type                     => "ManageIQ::Providers::Google::NetworkManager::LoadBalancerListener",
+      :load_balancer_protocol   => "TCP",
+      :instance_protocol        => "TCP",
+      :load_balancer_port_range => 61000...61002,
+      :instance_port_range      => 61000...61002
+    )
+    expect(lb.load_balancer_pools.first).to have_attributes(
+      :name    => "foo-lb",
+      :ems_ref => "7341375068641214585",
+      :type    => "ManageIQ::Providers::Google::NetworkManager::LoadBalancerPool",
+    )
+    expect(lb.load_balancer_pool_members.map { |m| m.vm.name })
+      .to include("instance-custom-machine-type").and include("wheezy")
```

Comment:
Would be nice to also test here, what happens when 1 VM is in multiple LBs. You should have 2 members pointing to 1 Vm then?